src/FusionAuthClient.ts

```diff
@@ -21,24 +21,36 @@ import ClientResponse from "./ClientResponse";
 
 export class FusionAuthClient {
   public clientBuilder: IRESTClientBuilder = new DefaultRESTClientBuilder();
+  public credentials: RequestCredentials;
 
-  constructor(public apiKey: string, public host: string, public tenantId?: string) {
-    this.apiKey = apiKey;
-    this.host = host;
-    this.tenantId = tenantId;
-  }
+  constructor(
+    public apiKey: string,
+    public host: string,
+    public tenantId?: string,
+  ) { }
```

Comment:
What was here before compiles to the following JavaScript:

```javascript
class DefaultRESTClient {
    constructor(apiKey, host, tenantId) {
        this.apiKey = apiKey;
        this.host = host;
        this.tenantId = tenantId;
        this.apiKey = apiKey;
        this.host = host;
        this.tenantId = tenantId;
    }
}
```

Reply:
Good catch.